src/test/java/com/iyzipay/sample/CheckoutFormSample.java

```diff
@@ -24,7 +26,10 @@ public void should_initialize_checkout_form() {
         request.setBasketId("B67832");
         request.setPaymentGroup(PaymentGroup.PRODUCT.name());
         request.setCallbackUrl("https://www.merchant.com/callback");
-        request.setDebitCardAllowed(Boolean.TRUE);
+
+        Map<String, String> metaData = new HashMap<String, String>();
+        metaData.put(PaymentCheckoutFormMetaDataKey.DEBIT_CARD_ALLOWED.name(), Boolean.TRUE.toString());
```

Comment:
After enum changes here we can put the key like: PaymentCheckoutFormMetaDataKey.DEBIT_CARD_ALLOWED.getValue()

Which return string of the metadata key in camelCase format as our standard for our parameters.